app/src/main/java/com/github/se/cyrcle/model/parking/Tile.kt

```diff
@@ -1,48 +1,56 @@
 package com.github.se.cyrcle.model.parking
 
+import android.annotation.SuppressLint
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 import com.mapbox.geojson.Point
 import com.mapbox.turf.TurfMeasurement
 import java.math.BigDecimal
 import java.math.RoundingMode
-import kotlin.math.pow
 
-const val TILE_SIZE = 0.1
-const val DECIMALS = 2
 /**
  * Represents a tile in the map.
  *
  * @param bottomLeft the bottom left corner of the tile
  * @param topRight the top right corner of the tile
+ * @param parkings the parkings in the tile
+ * @param uid the unique identifier of the tile
  */
-data class Tile(val bottomLeft: Point, val topRight: Point) {
+@Entity(tableName = "tiles")
+data class Tile(
+    val bottomLeft: Point,
+    val topRight: Point,
+    val parkings: MutableSet<Parking> = mutableSetOf(),
```

Comment:
Mhhhh good idea, I did not really think of that, one thing I might have to do is to expand the points into 4 doubles/floats to make the SQL query that gets in the rectangle since I cannot really deserialize the points in :star2: *SQL World* :star2: 
One think I could try doing is using the UID though since it encodes the position, I will have to look more into that in the later PR.